content/blog/2017-11-11-rperseus.md

```diff
@@ -0,0 +1,106 @@
+- r
+- API
+- packages
+- textmining
+- text-analysis
+- linguistics
+- data-extraction
+- community
+- software
+- review
+- onboarding
+authors:
+- name: David Ranzolin
+  url: https://github.com/daranzolin
+topicid: 1000
+---
+
+### Introduction
+
+When I was in grad school at Emory, I had a favorite desk in the library. The desk wasn’t particularly cozy or quiet, but what it lacked in comfort it made up for in real estate. My books and I needed room to operate. Students of the ancient world require many tools, and when jumping between commentaries, lexicons, and interlinears, additional clutter is additional “friction”, i.e., lapses in thought due to frustration. Technical solutions to this clutter exist, but the best ones are proprietary and expensive. Furthermore, they are somewhat inflexible, and you may have to shoehorn your thoughts into their framework. More friction.
+
+Interfacing with [the Perseus Digital Library](http://www.perseus.tufts.edu/hopper/) was a popular online alternative. The library includes a catalog of classical texts, a Greek and Latin lexicon, and a word study tool for appearances and references in other literature. If the university library’s reference copy of BDAG[^1] was unavailable, Perseus was our next best thing.
+
+[^1]: Bauer, Walter. *A Greek-English Lexicon of the New Testament and Other Early Christian Literature.* Edited by Frederick W. Danker. 3rd ed. Chicago: University of Chicago Press, 2000.
+
+Fast forward several years, and I’ve abandoned my quest to become a biblical scholar. Much to my father’s dismay, I’ve learned writing code is more fun than writing exegesis papers. Still, I enjoy dabbling with dead languages, and it was the desire to wed my two loves, biblical studies and R, that birthed my latest package, `rperseus`. 
+
+### Exploratory Data Analysis in Biblical Studies
+
+Working with the Perseus Digital Library was already a trip down memory lane, but here’s an example of how I would have leveraged `rperseus` many years ago.
+
+My best papers often sprung from the outer margins of my *Nestle-Aland Novum Testamentum Graece.* Here the editors inserted cross references to parallel vocabulary, themes, and even grammatical constructions. Given the intertextuality of biblical literature, the margins are a rich source of questions: Where else does the author use similar vocabulary? How is the source material used differently? Does the literary context affect our interpretation of this word? This is exploratory data analysis in biblical studies.     
+
+Unfortunately the excitement of the question is incommensurate with the tedium of the process--EDA continues by flipping back and forth between books, dog-earring pages, and avoiding paper cuts. `rperseus` aims to streamline this process with two functions: `get_perseus_text` and `perseus_parallel`. The former returns a data frame containing the text from any work in the Perseus Digital Library, and the latter renders a parallel in `ggplot2`. 
```

Comment:
Probably ok to spell out "exploratory data analysis" since phrase only used 3x, including once each in title and heading